gb-cpu/src/microcode/dec.rs

```diff
@@ -5,7 +5,7 @@ use super::{
 use crate::interfaces::WriteFlagReg;
 
 pub fn dec16(ctl: &mut MicrocodeController, _state: &mut State) -> MicrocodeFlow {
-    let value = ctl.pop_u16() + 1;
```

Comment:
This is also correcting a behaviour issue: `+ 1` instead of `- 1` for the decrement.